mne/viz/_3d.py

```diff
@@ -308,6 +310,16 @@ def plot_trans(info, trans='auto', subject=None, subjects_dir=None,
         If True, plot the digitization points.
     ref_meg : bool
         If True (default False), include reference MEG sensors.
+    ecog_sensors : bool
```

Comment:
Well we already have `meg_sensors` and `eeg_sensors`, so this is a bit more consistent. Do ECoG people never talk about sensors? Maybe they say "electrodes" but the consistency is a bit nice to have here, no?